src/Model.php

```diff
@@ -820,7 +820,8 @@ public function get(string $field = null)
                 $data[$field->short_name] = $this->get($field->short_name);
             }
 
-            return $data;
+            // return $data keys sorted in the order of Model::$only_fields
+            return array_merge($this->only_fields ? array_flip($this->only_fields): [], $data);
```

Comment:
array_merge? why to sort at all?

Reply:
Just for equivalence. Not sure if `only_fields` was not used to set the order in `get()` result.
Tests pass without the sorting so can be removed

Reply:
yes, return simply `$data`